Makefile

```diff
@@ -227,6 +233,12 @@ deploy: prepare ## Deploy controller to the K8s cluster specified in ~/.kube/con
 undeploy: prepare ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
 
+.PHONY: install-nowebhook uninstall-nowebhook
+install-nowebhook: prepare ## Install manifests without webhook configuration and operator deployment (for local run)
+	$(KUSTOMIZE) build config/nowebhook | kubectl apply --namespace $(OPERATOR_NAMESPACE) -f -
+uninstall-nowebhook: prepare ## Uninstall manifests deployed by install-nowebhook
+	$(KUSTOMIZE) build config/nowebhook | kubectl delete --ignore-not-found=$(ignore-not-found) -f -
```

Comment:
is this actually needed ? my understanding is that the `install` make target installs only the CRDs

Reply:
Let me recheck

Reply:
> is this actually needed ? my understanding is that the `install` make target installs only the CRDs

You are right. Will remove

Reply:
> > is this actually needed ? my understanding is that the `install` make target installs only the CRDs
> 
> You are right. Will remove

Was too fast. At least on uninstallation:
2024-10-09T12:08:04+03:00       ERROR   opendatahub.controllers.DataScienceCluster      failed to reconcile model-mesh on DataScienceCluster    {"instance.Name": "default-dsc", "error": "error to get rolebinding opendatahub from namespace opendatahub: RoleBinding.rbac.authorization.k8s.io \"opendatahub\" not found"}

Reply:
uninstall should only removes the CRDs (AFAIK), I guess here we are talking about the `deploy` and `undeploy` target ?

Reply:
I don't think there is an official documentation but more a convention on how the `Makefile` is generated by kubebuilder/operator-sdk, so if you run "make help" it should, among other things, output:

```
Deployment
  install          Install CRDs into the K8s cluster specified in ~/.kube/config.
  uninstall        Uninstall CRDs from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
  deploy           Deploy controller to the K8s cluster specified in ~/.kube/config.
  undeploy         Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
```

If the goal is to actually deploy the controller on k8s (as it seems), I think using  `deploy` as a prefix could be better for consistency, but I don't hold any strong opinion.

WDYT ?

Reply:
> 
> If the goal is to actually deploy the controller on k8s (as it seems), I think using `deploy` as a prefix could be better for consistency, but I don't hold any strong opinion.
> 
> WDYT ?

That's the point, the goal is _not_ to deploy the controller on k8s (but allow it run locally). That is why I used `install-*`

Reply:
But it seems the `kustomization.yaml` file also include `../default,` which actually [deploys the manager](https://github.com/opendatahub-io/opendatahub-operator/blob/6134496546f38dc1338623fb3f9688cf1efb332d/config/default/kustomization.yaml#L21) (AFAIK)

Reply:
Ok, let's remove that piece for now, does not look really important at the moment for local debugging